docs/gitrepo-structure.md

```diff
@@ -172,6 +183,40 @@ targetCustomizations:
     
     After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Make sure secret is created under the same namespace with gitrepo.
 
+### Using ValuesFrom
+
+These examples showcase the style and format for using `valuesFrom`.
+
+Example `ConfigMap`:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap-values
+  namespace: default
+data:  
+  values.yaml: |-
+    replication: true
+    replicas: 2
+    serviceType: NodePort
+```
+
+Example `Secret`:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-values
+  namespace: default
+stringData:
```

Comment:
We directly index `data` rather than `stringData`: https://github.com/rancher/fleet/blob/a1e504052b5fcd2e5112973acbe8024873cdff10/pkg/helmdeployer/deployer.go#L619

Having said that, I _do_ see a reason to allow the user to provide either or.

(side note: think that would a good idea? @StrongMonkey)

Reply:
@bashofmann explained to me offline:

> You can specify the data and/or the stringData field when creating a configuration file for a Secret. The data and the stringData fields are optional. The values for all keys in the data field have to be base64-encoded strings. If the conversion to base64 string is not desirable, you can choose to specify the stringData field instead, which accepts arbitrary strings as values.
> The keys of data and stringData must consist of alphanumeric characters, -, _ or .. All key-value pairs in the stringData field are internally merged into the data field. If a key appears in both the data and the stringData field, the value specified in the stringData field takes precedence.
>
> - https://kubernetes.io/docs/concepts/configuration/secret/#overview-of-secrets